cypress/integration/sidebar.spec.js/sidebar.spec.js

```diff
@@ -0,0 +1,95 @@
+/// <reference types="cypress" />
+
+describe('sidebar', () => {
+  // Based on sidebar.js
+  const defaultSectionName = 'getting-started';
+  const basepath = '/docs/2.0/'
+  beforeEach(() => {
+    // Takes to starting url which is baseURL
+    cy.visit(`${basepath}`)
```

Comment:
why interpolate?

Reply:
good catch! remnants of changing my mind :)